### PR TITLE
add `writeTimeline` functionality

### PR DESCRIPTION
**Pull request by ajaykarpur**
Writes a `Zorb_Timeline` protocol buffer to the UART RX characteristic.

### CODE REVIEW

File: SwiftyZorb/Public/ZorbDevice.swift [hunk 1]
```diff
@@ -421,6 +421,20 @@ final public class ZorbDevice {
         }
     }
     
+    /**
```
Comment (jakerockland) [inline SwiftyZorb/Public/ZorbDevice.swift:424]: This function needs a usage example. See `writeJavascript` as a model for how things should be formatted.

File: SwiftyZorb/Public/SwiftyZorb.swift [hunk 2]
```diff
@@ -228,6 +228,10 @@ public func writeJavascript(at url: URL, completion: @escaping WriteRequestCallb
     bluetoothManager.device?.writeJavascript(at: url) { result in completion(result) }
 }
 
+public func writeTimeline(_ timeline: Zorb_Timeline, completion: @escaping WriteRequestCallback) {
```
Comment (jakerockland) [inline SwiftyZorb/Public/SwiftyZorb.swift]: This function needs documentation. See public `writeJavascript` as a model for how things should be formatted.

**Review by jakerockland — CHANGES_REQUESTED** [review]

**Comment by jakerockland** [comment]
@ajaykarpur accompanying the changes requested above, the README needs to be updated to reflect the new functionality.

After making above documentation changes, new docs need to be generated with Jazzy. I'm creating a new PR that updated the README to note usage of Jazzy.

**Comment by jakerockland** [comment]
PR I described above: #47

File: SwiftyZorb/Public/ZorbDevice.swift (hunk 1 above)
Comment (ajaykarpur) [reply SwiftyZorb/Public/ZorbDevice.swift:424]: Added a usage example.

File: SwiftyZorb/Public/SwiftyZorb.swift (hunk 2 above)
Comment (ajaykarpur) [reply SwiftyZorb/Public/SwiftyZorb.swift]: Added documentation.

**Comment by ajaykarpur** [comment]
@jakerockland I've updated the README and added documentation for `writeTimeline`. I'm unable to generate new docs with `jazzy` (https://github.com/realm/jazzy/issues/1007).

**Review by jakerockland — APPROVED** [review]

**Pull request merged**